src/Facade/Services/SalesArticleService.cs

```diff
@@ -1,19 +1,25 @@
 ﻿namespace Linn.Products.Facade.Services
 {
     using System;
+    using System.Collections.Generic;
     using System.Linq;
     using System.Linq.Expressions;
+    using System.Security.Cryptography.X509Certificates;
```

Comment:
is this doing anything?

Reply:
Nope, no idea what I was trying that used that but removed now